pom.xml

```diff
@@ -48,6 +50,19 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
```

Comment:
No config added to connect to project on Sonar cloud.

Reply:
but config already added in pom in properties section to connect with sonar cloud